app/serialisers/services_serializer.rb

```diff
@@ -16,7 +16,7 @@ def attributes
     response = { services: all_services }
 
     response.tap do
-      response[:total_services] = Service.count if total_services.present?
+      response[:total_services] = (total_services.is_a?(Integer) ? total_services : Service.count) if total_services.present?
```

Comment:
then instead of passing a boolean to the serialise we change the initialiser to accept an integeger. defaulting to nil

`def initialize(services, total_services: nil)`

which means we can do something like:

`response[:total_services] = total_services if total_services.present?`

i think having the initialise argument be either an integer or nil is better than having the expectation of an integer or a boolean